lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -71,22 +71,23 @@ actor ConnectorSink is Sink
   """
   # ConnectorSink
 
-  `ConnectorSink` replaces the Pony standard library class `ConnectorConnection`
+  `ConnectorSink` replaces the Pony standard library class ConnectorConnection`
```

Comment:
I believe the ` should be present in `ConnectorConnection`

Reply:
good catch

Reply:
And actually there was a pre-existing problem there: `ConnectorConnection` should be `TCPConnection`.  Fixed.